client/reader/components/icons/a8c-conversations-icon.jsx

```diff
@@ -0,0 +1,17 @@
+export default function ReaderA8cConversationsIcon() {
+	return (
+		<svg
+			className="sidebar__menu-icon sidebar_svg sidebar_svg-a8c-conversations"
+			fill="none"
+			height="20"
+			viewBox="0 0 20 20"
+			width="20"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			<g stroke="#a2aab2" strokeWidth="1.5">
```

Comment:
Should we apply the colors through CSS instead of inlining it? We have dedicated classnames we could use for that.